src/partials/_sections.scss

```diff
@@ -0,0 +1,161 @@
+// styling of climbingForm Section
+.climbFormSection {
+    margin-bottom: 20px;
+    
+    .climbingForm {
+        h2 {
+            margin-bottom: 15px;
+            position: relative;
+        }
+        form {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            .dateArea {
+                width: 100%;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                background-color: $quaternary;
+                padding: 10px;
+                border-radius: 15px;
+                label {
+                    display: flex;
+                    margin-right: 30px;
+                    align-items: center;
+                }
+                input {
+                    cursor: pointer;
+                    background-color: $quaternary;
+                }
+            }
+    
+            .selectionArea {
+                margin: 10px 0 20px 0;
+                width: 100%;
+                display: flex;
+                flex-wrap: wrap;
+                
+    
+                .climbSelection {
+                    display: flex;
+                    flex-wrap: wrap;
+                    flex-direction: column;
```

Comment:
For accessibility add an outline, when tabbing through the forms user knows which form field they are on.